modules/ols-providing-custom-knowledge-to-the-llm.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="providing-custom-knowledge-to-the-llm_{context}"]
+= Providing custom knowledge to the LLM
+
+Customize the information available to the large language model (LLM) by providing access to a container image that resides in a remote image registry. The examples in this procedure use `quay.io` as the remote container image registry, and the path for the custom image is `quay.io/<username>/my-byok-image:latest`.
+
+:FeatureName: The BYO Knowledge feature
+include::snippets/technology-preview.adoc[]
```

Comment:
Should this go at the level of the concept? I am actually not sure if have an opinion on this as an org, but seems it might be good to know on first appearance. Although making sure they know before trying the steps also seems reasonable.

Reply:
@jeana-redhat I don't think we have an established norm for this. My thought was it might be more impactful to call this out with the instructions someone will use. I'm not sure people will always read the concept and then the task. If we call this out in the task, I think there is a better chance customers will see this when they perform the action.

I have another TP feature in the doc set up this way. If I change it here I should change that one too.

Reply:
yeah I am fine with it either way - keeping it close to the action works for me :)